src/Evolve/Dialect/Cassandra/CassandraKeyspace.cs

```diff
@@ -0,0 +1,167 @@
+﻿using Evolve.Connection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Evolve.Dialect.Cassandra
+{
+    public sealed class CassandraKeyspace : Schema
+    {
+        //Might want to make this configurable
+        private static ReplicationStrategy DefaultReplicationStrategy = CreateSimpleStrategy(1);
```

Comment:
Yes, you understood correctly. There are two possible strategies (at the moment) when you create a keyspace; in the case where Evolve would create the keyspace because it does not exist, it would be nice to be able to select which one as well as the parameters. I haven't thought yet on how to do that in a clean way.